package.json

```diff
@@ -134,6 +137,7 @@
     "valid-url": "1.0.9",
     "walk": "2.3.4",
     "webpack": "1.13.1",
+    "webpack-append": "0.1.2",
```

Comment:
`webpack-append` is so simple that I we might want to rewrite it ourselves :)

BTW, it should be called prepend!